### PR TITLE
2020 Date Change

### LINKED ISSUE #303: Update Green Up 2020 Date to May 30th
The date has changed for 5/2 to 5/30.
I assume this is only set in one place?

This should be seen:
- In the message when creating a new team
- In the change in the countdown days at the top of the home screen.

### PR DESCRIPTION
**Pull request by doub1ejack**
Updated the date logic & tests to handle the date change for May 30th, 2020.

**Home page banner**

**Create team date picker & info text**

### CODE REVIEW

File: libs/__tests__/green-up-day-calculators.test.js [hunk 1]
```diff
@@ -9,32 +11,55 @@ const addDays = (date, days) => new Date(date).setDate(new Date(date).getDate()
...
-        const myGUDay = getCurrentGreenUpDay(new Date("2019-05-07").toUTCString());
-        const expectedDateString = (new Date("2019-05-04T00:00:00.000Z")).toUTCString();
-        expect(myGUDay.toUTCString()).toBe(expectedDateString);
+        const myGUDay = getCurrentGreenUpDay(new Date("2019-05-07"));
+        const expectedDate = moment("2019-05-04").toDate();
+        expect(myGUDay).toMatchObject(expectedDate);
     });
+});
 
+describe("getNextGreenUpDay", () => {
```
Comment (doub1ejack) [inline libs/__tests__/green-up-day-calculators.test.js:30]: New tests for new `getNextGreenUpDay()` function

File: libs/__tests__/green-up-day-calculators.test.js [hunk 2]
```diff
@@ -9,32 +11,55 @@ const addDays = (date, days) => new Date(date).setDate(new Date(date).getDate()
 
 describe("getCurrentGreenUpDay", () => {
     it("returns this year's green up day if today if before Green Up Day", () => {
-        const myGUDay = getCurrentGreenUpDay(new Date("2019-01-01").toUTCString());
-        const expectedDateString = (new Date("2019-05-04T00:00:00.000Z")).toUTCString();
-        expect(myGUDay.toUTCString()).toBe(expectedDateString);
+        const myGUDay = getCurrentGreenUpDay(new Date("2019-01-01"));
+        const expectedDate = moment("2019-05-04").toDate();
+        expect(myGUDay).toMatchObject(expectedDate);
```
Comment (doub1ejack) [inline libs/__tests__/green-up-day-calculators.test.js:16]: Using `moment()` in libs/green-up-day-calculators.test.js means we're correctly using timezones and accounting for daylight savings time.  The hardcoded dates strings in the tests can't match those easily so I used `moment()` here as well as necessary.

File: libs/green-up-day-calucators.js [hunk 3]
```diff
@@ -2,34 +2,69 @@
...
+    return firstSaturdayInMay.toDate();
 };
 
-const addDays = (date: Date, days: number): number => new Date(date).setDate(new Date(date).getDate() + days);
+/**
+ * Returns the next Green Up day. If today is Green Up day, it returns today.
+ * @param today Date    An optional date used only by tests
+ * @returns Date        Date of the next Green Up day
+ */
+export function getNextGreenUpDay(today?: Date): Date {
```
Comment (doub1ejack) [inline libs/green-up-day-calucators.js:29]: This function is similar to `getCurrentGreenUpDay()` except that function will also return the past Green Up day if you are <=7 days past the date.  This one just returns the next Green Up day (accounting for the 2020 date shift).

**Review by doub1ejack — COMMENTED** [review]
Some comments for context.

**Review by johnneed — APPROVED** [review]

**Pull request merged**